motion/ruby_motion_query/subviews.rb

```diff
@@ -63,11 +62,14 @@ def add_subview(view_or_constant, opts={})
         new_view.rmq_appended if appended
 
         if self.stylesheet
-          apply_style_to_view(new_view, style) if style
+          apply_style_to_view(new_view, opts[:style]) if opts[:style]
```

Comment:
This was the only place the `style` above was used, so I just cleaned it up a tiny bit.